trace-utils/src/trace_utils.rs

```diff
@@ -526,6 +525,30 @@ pub fn enrich_span_with_mini_agent_metadata(
     }
 }
 
+pub fn enrich_span_with_google_cloud_function_metadata(
+    span: &mut pb::Span,
+    env_type: &EnvironmentType,
+) {
+    if let EnvironmentType::CloudFunction = env_type {
+        let function = span.meta.get("functionname");
+        let region = span.meta.get("gcrfx.location");
+        let project = span.meta.get("gcrfx.project_id");
+        if function.is_some() && region.is_some() && project.is_some() {
+            let resource_name = format!(
+                "projects/{}/locations/{}/functions/{}",
+                project.unwrap(),
+                region.unwrap(),
+                function.unwrap()
+            );
+
+            span.meta.insert(
+                "_dd.gcrfx.resource_name".to_string(),
```

Comment:
Is there a reason this resource span tag needs to be prefixed with `_dd`? For Azure App Services and Functions the resource name is visible to end users (ie. not hidden by using `_dd`).

Reply:
nevermind! After talking about it again in person, I updated the tag to exclude the `_dd` prefix